### ISSUE TITLE
[canary, @now/next] Cannot assign to read only property 'exports' of object '#<Object>'

### ISSUE THREAD

**Issue by joltmode** [zeit/next.js#5778]
# Bug report

## Describe the bug

When deploying Next app to Now with `@now/next`, I am getting error as of today:
```
Cannot assign to read only property 'exports' of object '#<Object>'
```

Seeing that `@now/next` uses Next.JS canary, I went to test it locally with canary release and can also replicate the issue.

Testing versions around, it seems to have been introduced with `7.0.2-canary.32` after: https://github.com/zeit/next.js/commit/55e89e759a1a81a2815cbcc53359197dbcb41779

I think it's due to the fact, that I am using both CJS and ES syntax for modules in my client, since some of the libraries are also used for backend in development.

## To Reproduce

Get the latest canary version of Next.JS and attempt to use CJS (`module.exports`) in your client codebase.

(Cannot set up example atm, am trying to extinguish this fire atm.)

## Expected behavior
Either documented behavior that this is getting changed and what we should do with our apps, or that it should work.

## Screenshots
`N/A`

## System information
 - OS: macOS
 - Browser (if applies): `N/A`
 - Version of Next.js: `7.0.2-canary.33` (the one that Now Next builder uses)

## Additional context

Is there a way to skip out of canary Next in Now (I suppose not, due to Lambdas) or use a specific version of canary?

Before further debugging, managed to ask on Spectrum: https://spectrum.chat/zeit/now/cannot-assign-to-read-only-property-exports-of-object-object~c701af9d-7d9c-4eda-8a07-35360003e19d

**Comment by timneutkens** [comment]
This is actually because @babel/runtime adds `import` statements when polyfilling. I guess we could check the source for `module.exports` and automatically apply `@babel/plugin-transform-modules-commonjs`.

**Comment by joltmode** [comment]
So I should manually add it now, to get it working?

Well, I tried to, but it seems to have messed with other stuff, now I'm getting errors about default export not being a function.

**Comment by timneutkens** [comment]
I'm trying to fix it right now, you could use the `"overrides"` key in babelrc to add the commonjs transform only for the specific files that are not esmodules as temporary workaround.

**Comment by joltmode** [comment]
Ok, got it running locally, deploying now to .... now.

**Comment by timneutkens** [comment]
Created a PR to fix it: #5780
